.github/workflows/pr.yml

```diff
@@ -182,7 +182,7 @@ jobs:
           mock-google-services: "true"
 
       - name: Run test
-        run: ./gradlew :common:testDebugUnitTest # Should be adapted once we have more than one tests
+        run: ./gradlew :common:testDebugUnitTest :app:testFullDebugUnitTest # Should be adapted once we have more than one tests
```

Comment:
The test is not full flavor specific. Is there a generic one or if not, can you add a note explaining why you put full here but not minimal?

Reply:
Currently we have 
> Candidates are: 'testDebugUnitTest', 'testFullDebugUnitTest', 'testFullReleaseUnitTest', 'testMinimalDebugUnitTest', 'testMinimalReleaseUnitTest', 'testReleaseUnitTest'.

Full and minimal are the same. I decided to not execute it for automotive since they are the same that's why I'm not using the top level calls.